### PR TITLE
WIP make Q and U doubles for issue #328 test

### PR DESCRIPTION
**Pull request by borg323**
This is just for testing #328.

### CODE REVIEW

**Comment by RedDenver** [comment]
This is a good way to figure out if the Q accuracy is even the cause of the 400k node scaling loss.

**Comment by oscardssmith** [comment]
Can someone please do testing here?

**Comment by hans-ekbrand** [comment]
> Can someone please do testing here?

I used this jupyter notebook to test it on google colab https://colab.research.google.com/drive/15KuHrdrDpqtL-rTXSJ_0j9bIa0eWUwvM#scrollTo=NJkXVgropChc

I got 47 games (100K per nodes, I believe) before I was disconnected. Master won, so PR349 was no improvement. At least not on this hardware, which I think is Tesla K80 (plain free google colab).

```
[...]
Started game 45 of 100 (lc0_master vs lc0_pr349)
Finished game 45 (lc0_master vs lc0_pr349): 1/2-1/2 {Draw by 3-fold repetition}
Score of lc0_master vs lc0_pr349: 7 - 5 - 33  [0.522] 45
Started game 46 of 100 (lc0_pr349 vs lc0_master)
Finished game 46 (lc0_pr349 vs lc0_master): 0-1 {Black wins by adjudication}
Score of lc0_master vs lc0_pr349: 8 - 5 - 33  [0.533] 46
Started game 47 of 100 (lc0_master vs lc0_pr349)
Finished game 47 (lc0_master vs lc0_pr349): 1-0 {White wins by adjudication}
Score of lc0_master vs lc0_pr349: 9 - 5 - 33  [0.543] 47
```

**Comment by borg323** [comment]
Please please the performance results on #328.

**Comment by hans-ekbrand** [comment]
> Please please the performance results on #328.

Aha, you didn't want #349 tested?

**Comment by borg323** [comment]
Thank you for reporting, all results are useful.

**Comment by borg323** [comment]
Closing this since #328 was closed as well.

**Comment by borg323** [comment]
There is renewed interest in this so I merged with master for testing.

**Comment by borg323** [comment]
Closing and reoppening trying to trigger a new appveyor build.

File: src/mcts/search.cc [hunk 1]
```diff
@@ -591,7 +591,7 @@ EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
   float max_n = 0.0;
   const float offset = params_.GetTemperatureVisitOffset();
   float max_eval = -1.0f;
```
Comment (MelleKoning) [inline src/mcts/search.cc]: max_eval also to double for precision comparison?

**Comment by borg323** [comment]
Closing this in favor of #878

**Pull request closed without merging**